libcoap/src/context.rs

```diff
@@ -406,6 +410,32 @@ impl CoapContext<'_> {
         self.add_endpoint(addr, coap_proto_t_COAP_PROTO_DTLS)
     }
 
+    // TODO: Possibly wrap in feature?
```

Comment:
As there is no way to disable multicast support in `libcoap` itself there is no advantage in adding a feature gate here (if libcoap always supports the feature, why shouldn't we?).